### ISSUE TITLE
Failed to create process: 0x00000002: The system cannot find the specified file.

### ISSUE THREAD

**Issue by Dachoni03** [vxiiduu/VxKex#84]
When I try to run Rufus on Windows 7 I'm getting the above error.

**Comment by CommanderLake** [comment]
Have a look at the windows event log and see if there are any more details in the application log.

**Comment by mtohky** [comment]
Reinstalling worked for me!

**Comment by Roadhog360** [comment]
I'm getting this too, reinstalling did not fix it.

@CommanderLake I opened the event log but I'm not sure what to look for, what should I do there?

**Comment by CommanderLake** [comment]
Just tested the latest Rufus on 7 and it opens without VxKex.

**Comment by Roadhog360** [comment]
I'm trying with SteelSeries GG

**Comment by asdf2345** [comment]
> I'm trying with SteelSeries GG

I laughed at this on your Reddit post.

![ScreenShot_20231217124207](https://github.com/vxiiduu/VxKex/assets/39643236/2b605bfe-a93f-4702-8be8-f029d8aea1aa)

**Comment by Roadhog360** [comment]
I love how he links to software someone later linked to me, which they claim is 7 compatible but clearly never tested because it is just an infinite white screen once installed and opened. Excellent work, "nerd".

**Comment by asdf2345** [comment]
I was curious so I tested 3.22.2, and it works on my machine.

**Comment by Roadhog360** [comment]
Weird. Honestly this is the end of my adventure. Trying to get this mouse's software working has just been one frustratingly cryptic issue after the next with seemingly no end in sight.

I would be happy to attempt to help test and continue attempting to open the latest version of the software on Windows 7 if my help is needed, but for now I'm going to just return my mouse and go with Logitech instead. I do have an old Rival 3 with a broken scroll wheel I can use to test anyone's given solutions if anyone else runs into the same issue as I do.

**Comment by asdf2345** [comment]
Yeah, I myself use an ancient Logitech mouse I've kept repairing, able to use SetPoint 5.42 with it. Best looking mouse software.

**Comment by ZV505867** [comment]
I have a similar issue, but with an Intel IME driver for Comet Lake. Sometimes, it would throw either error 0x00000002 or 0x0000007B error when attempting to run through VxKey.
